src/lib/y2network/connection_config/qeth.rb

```diff
@@ -47,14 +47,19 @@ class Qeth < Base
       attr_accessor :data_channel
       # @return [Boolean] whether layer2 is enabled or not
       attr_accessor :layer2
+      # @return [Boolean] whether ipa takeover is enabled or not
+      attr_accessor :ipa_takeover
       # @return [Integer] port number (0 or 1)
       attr_accessor :port_number
+      # @return [String] configuration extra attributes
+      attr_accessor :attributes
```

Comment:
please try to merge network-ng and add it to `==`

Reply:
Rebased